src/math/mod.rs

```diff
@@ -4,4 +4,5 @@ pub mod utils;
 
 mod rect;
 pub use self::rect::Rect;
-pub(crate) use self::utils::resize_dimensions;
+
+pub use self::utils::{resize_to_fit, resize_to_fill};
```

Comment:
I'd just implement them here instead of in `utils` and not re-export them to have them visible in two places.